static/examples/cstg-prebid-example/README.md

```diff
@@ -12,4 +14,4 @@ When running locally, the configuration values in `index.html` will not work, as
 
 ## Prebid.js
 
-This file is a build of Prebid.js for the CSTG example, with the userId, uid2IdSystem, appnexusBidAdapter modules included.
\ No newline at end of file
+This file is a build of Prebid.js with the userId, uid2IdSystem, appnexusBidAdapter modules included.
```

Comment:
This list of three modules -- should we have an "and" or an "or" in there? Just checking. Can't tell.

Reply:
it's "and". fixed